test/test_va_api_fixture.cpp

```diff
@@ -113,8 +120,46 @@ void VAAPIFixture::doGetMaxEntrypoints()
 void VAAPIFixture::doGetMaxNumConfigAttribs()
 {
     m_maxConfigAttributes = vaMaxNumConfigAttributes(m_vaDisplay);
+    ASSERT_GT(m_maxConfigAttributes, 0);
```

Comment:
EXPECT_GT is probably better here and other similar cases below since ASSERT* in subroutines only abort the current function, not the test... see [GoogleTest Advanced Guide](https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#propagating-fatal-failures) for more details about this.

Reply:
Fixed.